### PR TITLE
simple 404 page

### PR DESCRIPTION
**Pull request by NathHorrigan**

### CODE REVIEW

File: src/pages/404/index.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+export default ({ data }) => {
+    return <NotFoundPage contact={data.wagtail.contact} contactReasons={data.wagtail.contactReasons} />
+}
+
+export const query = graphql`
+  query {
+    wagtail {
+      contact {
+        ...contactSnippet
```
Comment (helenb) [inline src/pages/404/index.js]: how do editors add contact reasons on the 404 page in wagtail?

File: src/pages/404/index.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      }  
+      contactReasons {
+        ...contactReasonsSnippet
+      }
+    }
+  }
+`
```
Comment (helenb) [inline src/pages/404/index.js]: add a new line

**Comment by helenb** [comment]
Just a query and a v small point.

File: src/pages/404/index.js (hunk 1 above)
Comment (NathHorrigan) [reply src/pages/404/index.js]: In Slack we said that content isn't supposed to be editable, for now the page grabs the default snippets.

**Pull request merged**